public/acts/act3-4/isaquest.html

```diff
@@ -2507,7 +2506,7 @@ <h4>ACT 4</h4>
         (You smile back.)
     </p>
 
-    <br>
+    <hr>
 
     <img src="https://instarsandtime.wiki.gg/images/5/55/ISAT_Event_ACT3_Friendquest_Snack_Ending_1.png" alt="The entire group having a picnic. Everyone is happy and smiling while huddled around a variety of food. Bonnie triumphantly holds up a plate of samosas.">
```

Comment:
oh, something i've been doing is adding "image ID:" and "end ID." to the starts and ends respectively of alt text to make it clearer when a screenreader's reading out an image vs normal text. i don't know if any screenreaders announce "this is an image", but windows narrator doesn't at least.

(eg. `<img src="source" alt="image ID: alt text here. end ID.">`. "ID" needs to be capitalized to be read out as an acronym)